oauth2/provider/login.php

```diff
@@ -31,8 +32,10 @@
                 <div class="card-body">
                     <?php if (empty($authorize) && !$mfaRequired) { ?>
                         <h4 class="card-title mb-4 mt-1"><?php echo xlt("Sign In"); ?></h4>
-                    <?php } elseif (empty($mfaRequired) && $mfaRequired) { ?>
+                    <?php } elseif (empty($mfaRequired) && $mfaRequired && $mfaType === MfaUtils::TOTP) { ?>
                         <h4 class="card-title mb-4 mt-1"><?php echo xlt('TOTP Verification'); ?></h4>
+                    <?php } elseif (empty($mfaRequired) && $mfaRequired && $mfaType === MfaUtils::U2F) { ?>
+                        <h4 class="card-title mb-4 mt-1"><?php echo xlt('U2F Key Verification'); ?></h4>
```

Comment:
empty($mfaRequired) && $mfaRequired will always be false won't it. Also, not PHP8 compatible e.g  empty($mfaRequired) && !empty($mfaRequired) or empty($mfaRequired) && $mfaRequired === ?.

Reply:
Done